pkg/clusterapi/system.go

```diff
@@ -97,6 +122,18 @@ func (c *system) Run(ctx context.Context) error { //nolint:gocyclo
 
 	// Create the local control plane.
 	lcp := &localControlPlane{}
+
+	ipv4, err := hostHasIPv4Address()
+	if err != nil {
+		return err
+	}
+	// If the host has no IPv4 available, the default value of service network should be modified to IPv6 CIDR.
+	if !ipv4 {
+		lcp.APIServerArgs = map[string]string{
+			"service-cluster-ip-range": "fd02::/112",
```

Comment:
I'm still unsure if the value I chose for the CIDR is the ideal one.
But when I tested it worked without issues.

Reply:
`/112` is quite a big subnet, usually the ipv6 subnets are in `/64` and that's still enormous. Are we using `/112` anywhere else in the installer?

Reply:
I took it from [here](https://github.com/openshift/installer/blob/main/upi/openstack/inventory.yaml#L77).
But we can change the value if needed.

Reply:
Honestly, I don't know. As we won't be creating any services, I'm not sure how much this range will matter. So I'm fine with either this value or the suggested one, which makes sense.

Reply:
Yep it looks like this range does not really matter as it's all internal for the installer.